tests/admin_views/tests.py

```diff
@@ -1974,6 +2051,18 @@ def test_has_module_permission(self):
         self.assertContains(response, 'admin_views')
         self.assertContains(response, 'Articles')
 
+    def test_has_module_view_permission(self):
+        """
+        Ensure that has_module_permission() returns True for all users who
+        have view permission for that module, so that
+        the module is displayed on the admin index page.
+        """
+        self.client.login(**self.viewuser_login)
+        response = self.client.get(self.index_url)
+        self.assertContains(response, 'admin_views')
+        self.assertContains(response, 'Articles')
+        self.client.logout()
```

Comment:
no need to logout at the end of a test -- this is done automatically for each test